profiler/src/main/java/com/splunk/opentelemetry/profiler/Configuration.java

```diff
@@ -30,6 +30,7 @@
       "splunk.profiler.recording.duration";
   public static final String CONFIG_KEY_KEEP_FILES = "splunk.profiler.keep-files";
   public static final String CONFIG_KEY_INGEST_URL = "splunk.profiler.logs-endpoint";
+  public static final String CONFIG_KEY_PERIOD_PREFIX = "splunk.profiler.period";
```

Comment:
why not put the `.` at the end of this, and not have to append it inline?

Reply:
It was intentional, and purely preference. 

I used directory/paths as a reference...and it's miserable to always guess whether or not the first part has a trailing slash or not. (My) preference is always given to omitting slash from the dir and always having the slash present when concatenating because it feels more explicit.

Reply:
cool. one other thing...do these values need to be public?